### DISCUSSION TITLE
Update-2022-10-17

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#141] [Announcements]
Implemented `voxel-grid-shape` as a way of rendering `isosurface` shapes. Demo in `demo-isosurface.lisp`

Animating isosurface threshold:

https://user-images.githubusercontent.com/103860793/196287428-e4f0e204-902e-4180-9128-d5584d64cf46.mov

**Comment by aykaramba** [comment]
This is amazing.
